tests/realtime/realtimeUtils.js

```diff
@@ -132,6 +141,18 @@ describe('tests/realtime/realtimeUtils.js, realtime utils Tests >', () => {
         expect(realtimeUtils.shouldIEmitThisObj(nspString,
           looksLikeSampleObjNAUS)).to.equal(true);
       });
+
+      it('for should return true for roomTest', () => {
+        const nspString = realtimeUtils.getNamespaceStringBots(roomTest);
+        expect(realtimeUtils.shouldIEmitThisObj(nspString,
```

Comment:
`expect(realtimeUtils.shouldIEmitThisObj(nspString, roomTest))` should all stay on one line together
`.to.equal(...)` should be on next line